src/core/thread/osthread.d

```diff
@@ -1961,7 +1961,8 @@ extern (C) void thread_init() @nogc
     // The Android VM runtime intercepts SIGUSR1 and apparently doesn't allow
     // its signal handler to run, so swap the two signals on Android, since
     // thread_resumeHandler does nothing.
-    version (Android) thread_setGCSignals(SIGUSR2, SIGUSR1);
+    // This hack is no longer needed, as the GC signals now use SIGRTMIN / SIGRTMIN + 1
+    // version (Android) thread_setGCSignals(SIGUSR2, SIGUSR1);
```

Comment:
I'd say remove this altogether, it'll probably just be confusing and irrelevant in the future.